README.md

```diff
@@ -60,7 +60,7 @@ repo-token | `secrets.GITHUB_TOKEN` | Access token for which this action will ru
 ignore-authors | dependabot<br/>dependabot[bot] | If the action detects that the PR author is one of these logins, it will skip checks and set all outputs to `true`.
 base-comment | (see [action.yml](./action.yml)) | Preamble to any comment the action leaves on the PR.
 ignore-team-members | true | Whether to ignore in the case that the PR author is a) the owner of repository, or b) has a public membership[^1] in the org that owns the repository.
-body-regex | `(fix\|resolv\|clos)(e)*(s\|d)* #\d+` | Regular expression to identify whether the PR body refers to an issue[^2][^3].
+body-regex | `(fix(es|ed)?|((resolve|close)(s|d)?)) #\d*[1-9]\d*` | Regular expression to identify whether the PR body refers to an issue[^2][^3].
```

Comment:
It looks like there needs to be some escaping to let the markdown table render correctly. Note the pipe characters are escaped in the original because they represent table borders.

Preview is here, note the table row for `body-regex`: https://github.com/mtfoley/pr-compliance-action/blob/7e77ac91d12627072e794fbce718e4dc82036086/README.md

Reply:
fixed